OMX-27-firmware/OMX-27-firmware.ino

```diff
@@ -27,6 +27,21 @@
 #include "scales.h"
 #include "util.h"
 
+typedef struct {
+	const char* label;
+	int* valuePtr;
+	bool readOnly;
+	int min, max;
+	void (*drawValue)(int,int);
+	void (*onChange)(int*,int,int);
+	char* (*getLabel)();
+	int (*getValue)();
+} ui_param;
```

Comment:
Nice, using function pointers is a nice way to decouple behaviour from the display logic.